lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -309,15 +309,18 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
     return NI.allInputsAndOutputsHaveSameElemKind({ElemKind::Int8QTy});
 
   case Kinded::Kind::ConvertToNodeKind:
-    return ((NI.getInElemTy(ConvertToNode::InputIdx) == ElemKind::FloatTy) &&
-            ((NI.getOutElemTy(ConvertToNode::ResultIdx) ==
-              ElemKind::Float16Ty) ||
-             (NI.getOutElemTy(ConvertToNode::ResultIdx) ==
-              ElemKind::Int64ITy))) ||
-           ((NI.getInElemTy(ConvertToNode::InputIdx) == ElemKind::Float16Ty) &&
-            (NI.getOutElemTy(ConvertToNode::ResultIdx) == ElemKind::FloatTy)) ||
-           ((NI.getInElemTy(ConvertToNode::InputIdx) == ElemKind::Int64ITy) &&
-            (NI.getOutElemTy(ConvertToNode::ResultIdx) == ElemKind::FloatTy));
+    return (((NI.getInElemTy(ConvertToNode::InputIdx) == ElemKind::FloatTy) ||
+             (NI.getInElemTy(ConvertToNode::InputIdx) ==
+              ElemKind::Float16Ty)) &&
+            ((NI.getInElemTy(ConvertToNode::ResultIdx) == ElemKind::FloatTy) ||
```

Comment:
Does not seem correct. ConvertToNode::ResultIdx - this is output of the node. NI.getInElemTy() checks for inputs.

Reply:
Err, you're right. Good catch!